api/proto/metrics/metrics.proto

```diff
@@ -144,6 +145,27 @@ message appMetric {
   repeated appDiskMetric disk = 6;
 }
 
+message logMetric {
+  uint64 numDeviceEventsSent = 1;
+  uint64 numDeviceBundlesSent = 2;
+  uint64 numAppEventsSent = 3;
+  uint64 numAppBundlesSent = 4;
+  uint64 num4xxResponses = 5;
+  google.protobuf.Timestamp lastDeviceBundleSendTime = 6;
+  google.protobuf.Timestamp lastAppBundleSendTime = 7;
+
+  bool isLogProcessingDeferred = 8;
+  uint64 numTimesDeferred = 9;
+  google.protobuf.Timestamp lastLogDeferTime = 10;
+
+  uint64 avgBundleLogCount = 11;
+  uint64 avgBundleProtoByteCount = 12;
+  uint64 numBundleOverFlows = 13;
```

Comment:
For device? For apps?
Does it count the number of events dropped something else?

Is there some other reason we'd drop events?

Reply:
These bundles are not discarded, but instead they are immediately sent to cloud. This counter incrementing hints that there is lot of incoming pressure from rsyslogd.

Reply:
You mean that a bundle is sent either because of a timeout or that it has reached a max count or max bytes. Is this counting when it hit the max count or bytes?

I don't if that tell much about load.

Reply:
This counter increments when the bundle exceeds either max event count or max byte count. If it is the 10s timer that exports bundles, it kind of indicates that the log inflow is slow. But, if bundle overflows happen, it may indicate that there is a constant inflow/ rapid inflow of logs from rsyslogd.